features/step_definitions/networking.rb

```diff
@@ -628,7 +628,7 @@
       pod.node_name == node_name
     }.first
     cache_resources ovnkube_pod
-    @result = ovnkube_pod.exec(network_cmd, as: admin)
+    @result = ovnkube_pod.exec(network_cmd, container: "ovn-controller", as: admin)
```

Comment:
I forgot they added `kube-rbac-proxy` container to `ovnkube-node` pod as well.  So we need to specify the container. :+1: